runtime/runtime/src/ext.rs

```diff
@@ -379,11 +379,12 @@ impl<'a> Contract for RuntimeContractExt<'a> {
         let code_hash = self.hash();
         let version = self.current_protocol_version;
         let chain_id = self.chain_id;
+        // We are removing the check to match code_hash_matches_wallet_contract
+        // If the account type is EthImplicitAccount, we always load the latest (PV 71) wallet contract.
         if checked_feature!("stable", EthImplicitAccounts, version)
             && account_id.get_account_type() == AccountType::EthImplicitAccount
-            && code_hash_matches_wallet_contract(chain_id, &code_hash, version)
```

Comment:
todo: Verify if it is fine to remove this check

Reply:
We actually cannot remove this check because not all accounts that look like eth-implicit have the wallet contract magic bytes (see details in [this PR](https://github.com/near/nearcore/pull/11606) description).